build_all_images.sh

```diff
@@ -688,7 +603,35 @@ function validate_bundles_yml {
 }
 
 function _build_jdk {
-	echo "TODO"
+	local jdk_friendly_name="${1}"
+	local jdk_image_name="${2}"
+	local jdk_version="${3}"
+
+	local latest_available_zulu_amd64_version=$(get_latest_available_zulu_version "${jdk_version}" "amd64")
+	local latest_available_zulu_arm64_version=$(get_latest_available_zulu_version "${jdk_version}" "arm64")
+
+	if [[ $(get_latest_docker_hub_zulu_version "${jdk_image_name}" "${jdk_version}" "amd64") == "${latest_available_zulu_amd64_version}" ]] && [[ $(get_latest_docker_hub_zulu_version "${jdk_image_name}" "${jdk_version}" "arm64") == "${latest_available_zulu_arm64_version}" ]] && [[ "${LIFERAY_DOCKER_DEVELOPER_MODE}" != "true" ]]
+	then
+		echo ""
+		echo "Docker image ${jdk_friendly_name} is up to date."
+
+		return
+	fi
+
+	echo ""
+	echo "Building Docker image ${jdk_friendly_name}."
+	echo ""
+
+	LIFERAY_DOCKER_IMAGE_PLATFORMS="${LIFERAY_DOCKER_IMAGE_PLATFORMS}" LIFERAY_DOCKER_REPOSITORY="${LIFERAY_DOCKER_REPOSITORY}" LIFERAY_DOCKER_ZULU_11_AMD64_VERSION=${latest_available_zulu_amd64_version} LIFERAY_DOCKER_ZULU_11_ARM64_VERSION=${latest_available_zulu_arm64_version} time ./build_"${jdk_image_name//-/_}"_image.sh "${BUILD_ALL_IMAGES_PUSH}" | tee -a "${LIFERAY_DOCKER_LOGS_DIR}"/"${jdk_image_name}".log
```

Comment:
@Wykenz, there are variables still referencing the Zulu 11, like `LIFERAY_DOCKER_ZULU_11_ARM64_VERSION`. Can you please fix it?

Please see how the variable should be named in https://github.com/natocesarrego/liferay-docker/pull/82/commits/dcce0055420e8314aff997cbee278abe7aebbabe#

Reply:
zulu_amd64_version="LIFERAY_DOCKER_ZULU_${jdk_version_number}_AMD64_VERSION"